src/NexusMods.App.UI/Windows/IMainWindowViewModel.cs

```diff
@@ -1,8 +1,20 @@
+using System.Reactive;
 using NexusMods.App.UI.Overlays;
+using ReactiveUI;
 
 namespace NexusMods.App.UI.Windows;
 
 public interface IMainWindowViewModel : IViewModelInterface, IWorkspaceWindow
 {
     IOverlayViewModel? CurrentOverlay { get; }
+
+    /// <summary>
+    ///     This command is used to bring the window to front.
+    /// </summary>
+    /// <remarks>
+    ///     Note(sewer)
+    ///     Normally this would go into something like <see cref="IWorkspaceWindow"/>,
+    ///     however there aren't enough use cases yet to justify this.
+    /// </remarks>
+    ReactiveCommand<Unit, Unit> ActivateWindowCommand { get; }
```

Comment:
`ActivateWindow` is a terrible name. Call it what it is: `BringWindowToFront`.

This command should also go into the `IWorkspaceWindow` interface.